lib/switchUserAccounts.css

```diff
@@ -0,0 +1,9 @@
+.chaise-body li {
```

Comment:
Let's add this to the `app.scss` that we have. You can create a different SCSS file and then import it in the `app.scss` but we shouldn't create separate CSS files anymore.